runelite-client/src/main/java/net/runelite/client/plugins/examine/ExaminePlugin.java

```diff
@@ -197,76 +201,67 @@ private void findExamineItem(PendingExamine pendingExamine)
 		int widgetChild = TO_CHILD(widgetId);
 		Widget widget = client.getWidget(widgetGroup, widgetChild);
 
-		if (widget == null)
```

Comment:
Because when you examine something on the ground the `widget` object is always gonna come up null because of all of that widgetId magic going on right above it. Theres no mapping to an appropriate Widget because widgetID you get back when examining random stuff in the world/on the ground could be any number of values. 

In the event that you `pendingExamine.getType() == ExamineType.ITEM`, you should be able to get the price lookups but that guard is preventing that.